.github/workflows/project-check.yml

```diff
@@ -111,6 +103,24 @@ jobs:
       run: |
         echo "Pass log is ${{ env.PASS_LOG }}"
 
+    - name: Generate badge
```

Comment:
As we discussed workflow for badges should not be part of project checks. We can trigger it after we merge PR for project checks

Reply:
I waned to keep it for the automation